app/src/androidTest/java/org/mozilla/fenix/ui/CollectionTest.kt

```diff
@@ -42,31 +54,152 @@ class CollectionTest {
         mockWebServer.shutdown()
     }
 
-    @Ignore
     @Test
+    // open a webpage, and add currently opened tab to existing collection
     fun AddTabToCollectionTest() {
```

Comment:
All looks great, just a few  style nits.... for consistency, all function names should prob begin w/ lower case